src/wms/src/main/java/org/geoserver/wms/WMS.java

```diff
@@ -218,6 +218,12 @@
     /** DateLine Wrapping Heuristic key */
     public static String DATELINE_WRAPPING_HEURISTIC_KEY = "disableDatelineWrappingHeuristic";
 
+    /** Capabilities will be produced without a root Layer element, when not needed * */
+    public static Boolean NO_ROOT_LAYER_IN_CAPABILITIES = false;
```

Comment:
"NO_ROOT_LAYER_IN_CAPS_DEFAULT" maybe (was not clear to me reading the code that it was the default value, tried to make the name a bit shorter to compensate)